### DISCUSSION TITLE
CT2 address

### DISCUSSION THREAD

**Discussion by delibury** [StealthChesnut/HA-FoxESS-Modbus#11] [Q&A] [answered]
I've got E3000 solar system fitted on the green energy grant and recently added a AC1 inverter and batteries. 
I've got home assistant running on a raspberry pi4 and I'm monitoring the battery system via serial to RS485 converter and HA-FoxESS-Modbus.

I'm trying to get data from the E3000 solar inverter, I've tried scanning the RS485 connection on the inverter with mbpoll and as far as I can tell it's acting as a master and scanning for a meter.

So I've bought a second CT and connected it to the CT2 input of the AC1 inverter, does anyone know the modbus address?
I'm checking every address from 11000 to 11040 in home assistant but having no luck.

**Comment by williamjeccles** [comment]
Good question, @StealthChesnut any ideas what this might be?

**Comment by StealthChesnut** [comment]
@delibury : can you describe in a bit more detail what exactly you are trying to do (and why)?
What is an E3000?

It sounds like you're trying to get the AC1 to read the power input/output of the E3000 by using a CT clamp, connected to the CT2 input on the AC1 - is that correct? Is the E3000 supplying power to the system and therefore the AC1 needs to know about it?

The data registers go up to 11095, so there's plenty more to check/monitor! I have zeros in the following registers and nothing connected to CT2, so if you have values you might try looking at them:
(all starting 110xx)
26
27
30
31
51
53
55
60-69
72
75
78
80
90

Do you have anything in any of those?

**Reply by delibury** [reply]
Hi the E3000 is the Solar inverter fitted under the green energy deal, It's a Fox inverter and looks exactly the same as a Fox F3000.
The AC1 doesn't need to know the output of the E3000 it just uses the grid CT.
I'm trying to get the solar production data for the energy dashboard,
I'm getting 89 on 11080 that could be it as my my inverters generating 877 watts. I'll keep an eye on it and let you know.

**Comment by delibury** [comment, marked as answer]
After some testing I found that 11022 is CT2.
I now have my energy dashboard working.
Thankyou.

**Comment by williamjeccles** [comment]
I've added this to the main project under commit & merge [williamjeccles-add-CT2-support](https://github.com/StealthChesnut/HA-FoxESS-Modbus/pull/29)

**Comment by williamjeccles** [comment]
Also found and added CT2 for LAN modbus [Added-CT2-for-LAN](https://github.com/StealthChesnut/HA-FoxESS-Modbus/pull/83)
